alpine/base-requirements.in

```diff
@@ -0,0 +1,35 @@
+# WSGI
+gunicorn
+gevent
+
+# Flask and related plugins
+flask
+flask-cors
+flask-apispec
+
+# Other common dependencies
+raven[flask]
+python-jose
+
+# Development (QA, testing, debugger tools)
+pytest
+pytest-cov
+black
+flake8
+flake8-docstrings
+flake8-bugbear
+isort
+safety
+ipython
+
+# Compatibility constraints
+
+# pydocstyle (dependency of flake8-docstrings): pin to v3.0.0 until
+# flake8-docstrings fixes compatibility with v4.0.0
+pydocstyle==3.0.0
+
+# apispec and marshmallow (dependencies of flask-apispec): Upgrading to latest
```

Comment:
Kinda sad that flask-apispec doesn't pin them itself :/

Reply:
Well, we haven't really investigated why the latest versions break. Could very well be a backwards incompatibility that we just need to address in our code. But yes if they're simply incompatible, that would be sad!